src/main/java/com/google/sps/python/heatMap.py

```diff
@@ -0,0 +1,186 @@
+import numpy as np
+import json
+_POPULATION = 'datasets/county-population.txt'
+_CASES = 'datasets/by-county-and-date.txt'
+_COORDINATES = 'datasets/usa_county_wise.txt'
+_WADE_HAMPTON_CENSUS_AK = 96
+_GRAND_PRINCESS_CRUISE_CA = 193
+_NEW_YORK_CITY_UNALLOCATED = 1864
+_IGNORE_LOCATIONS = [_WADE_HAMPTON_CENSUS_AK, _GRAND_PRINCESS_CRUISE_CA, _NEW_YORK_CITY_UNALLOCATED]
+_US_STATE_ABBREV = {
+    'Alabama': 'AL',
+    'Alaska': 'AK',
+    'American Samoa': 'AS',
+    'Arizona': 'AZ',
+    'Arkansas': 'AR',
+    'California': 'CA',
+    'Colorado': 'CO',
+    'Connecticut': 'CT',
+    'Delaware': 'DE',
+    'District of Columbia': 'DC',
+    'Florida': 'FL',
+    'Georgia': 'GA',
+    'Guam': 'GU',
+    'Hawaii': 'HI',
+    'Idaho': 'ID',
+    'Illinois': 'IL',
+    'Indiana': 'IN',
+    'Iowa': 'IA',
+    'Kansas': 'KS',
+    'Kentucky': 'KY',
+    'Louisiana': 'LA',
+    'Maine': 'ME',
+    'Maryland': 'MD',
+    'Massachusetts': 'MA',
+    'Michigan': 'MI',
+    'Minnesota': 'MN',
+    'Mississippi': 'MS',
+    'Missouri': 'MO',
+    'Montana': 'MT',
+    'Nebraska': 'NE',
+    'Nevada': 'NV',
+    'New Hampshire': 'NH',
+    'New Jersey': 'NJ',
+    'New Mexico': 'NM',
+    'New York': 'NY',
+    'North Carolina': 'NC',
+    'North Dakota': 'ND',
+    'Northern Mariana Islands':'MP',
+    'Ohio': 'OH',
+    'Oklahoma': 'OK',
+    'Oregon': 'OR',
+    'Pennsylvania': 'PA',
+    'Puerto Rico': 'PR',
+    'Rhode Island': 'RI',
+    'South Carolina': 'SC',
+    'South Dakota': 'SD',
+    'Tennessee': 'TN',
+    'Texas': 'TX',
+    'Utah': 'UT',
+    'Vermont': 'VT',
+    'Virgin Islands': 'VI',
+    'Virginia': 'VA',
+    'Washington': 'WA',
+    'West Virginia': 'WV',
+    'Wisconsin': 'WI',
+    'Wyoming': 'WY'
+}
+
+
+# Loads data from all three datasets above and returns a
+# python dict that maps a county's name to its coordinates
+# and a covid risk weight based on recency of outbreak
+# at that locationa and that county's population
+def load_covid_data():
+    county_to_coordinates = read_county_wise()
+    population_map = load_population_map(county_to_coordinates)
+    weighted_cases = load_weighted_cases(population_map)
+    i = 0
+    for county in population_map:
+        population_map[county]["weight"] = int(weighted_cases[i]
+          / population_map[county]["weight"]) # Normalizes weights by population density.
+        i += 1
+    return population_map
+
+
+# Reads file to create a python dict that
+# maps county names to their coordinates.
```

Comment:
Would be helpful to have the line format in the comment, so it's easier to understand what the function is doing. Here and in the other functions that read files.

// A sample line from the file will have the following structure:
// UID,iso2,iso3,code3,FIPS,Admin2,Province_State,Country_Region,Lat,Long_,Combined_Key,Date,Confirmed,Deaths
//